osu.Game/Localisation/GeneralSettingsStrings.cs

```diff
@@ -79,23 +79,31 @@ public static class GeneralSettingsStrings
         /// </summary>
         public static LocalisableString LearnMoreAboutLazerTooltip => new TranslatableString(getKey(@"check_out_the_feature_comparison"), @"Check out the feature comparison and FAQ");
 
+        /// <summary>
+        /// "Check with your package manager / provider for other release streams."
+        /// </summary>
+        public static LocalisableString ChangeReleaseStreamPackageManagerWarning => new TranslatableString(getKey(@"change_release_stream_package_warning"), @"Check with your package manager / provider for other release streams.");
+
+        /// <summary>
+        /// "Check with your app store (testflight, etc) for other release streams."
+        /// </summary>
+        public static LocalisableString ChangeReleaseStreamMobileWarning => new TranslatableString(getKey(@"change_release_stream_mobile_warning"), @"Check with your app store (testflight, etc) for other release streams.");
+
         /// <summary>
         /// "Are you sure you want to run a potentially unstable version of the game?"
         /// </summary>
-        public static LocalisableString ChangeReleaseStreamConfirmation => new TranslatableString(getKey(@"change_release stream_confirmation"),
-            @"Are you sure you want to run a potentially unstable version of the game?");
+        public static LocalisableString ChangeReleaseStreamConfirmation => new TranslatableString(getKey(@"change_release_stream_confirmation"), @"Are you sure you want to run a potentially unstable version of the game?");
 
         /// <summary>
         /// "If you run into issues starting the game, you can usually run the installer from the official site to recover."
         /// </summary>
-        public static LocalisableString ChangeReleaseStreamConfirmationInfo => new TranslatableString(getKey(@"change_release stream_confirmation_info"),
-            @"If you run into issues starting the game, you can usually run the installer from the official site to recover.");
+        public static LocalisableString ChangeReleaseStreamConfirmationInfo => new TranslatableString(getKey(@"change_release_stream_confirmation_info"), @"If you run into issues starting the game, you can usually run the installer from the official site to recover.");
```

Comment:
I've changed the keys from `change_release stream_confirmation[_info]` to `change_release_stream_confirmation[_info]` (added underscore). It's a small change, however I believe the former keys are invalid given there are no exported localisations:

```sh
$ rg "change_release"
osu.Game.Resources/Localisation/GeneralSettings.resx
103:  <data name="change_release stream_confirmation" xml:space="preserve">
106:  <data name="change_release stream_confirmation_info" xml:space="preserve">
```

(that one is the EN localisation generated by the `dotnet` tool)